src/sql/src/plan/transform_ast.rs

```diff
@@ -454,10 +454,10 @@ impl<'a> FuncRewriter<'a> {
             Expr::Identifier(ident) if ident.len() == 1 => {
                 let ident = normalize::ident(ident[0].clone());
                 let fn_ident = match ident.as_str() {
-                    "current_role" => Some("current_user"),
-                    "current_schema" | "current_timestamp" | "current_user" | "session_user" => {
-                        Some(ident.as_str())
-                    }
+                    "current_role" | "user" => Some("current_user"),
+                    "current_catalog" => Some("current_database"),
+                    "current_schema" | "current_timestamp" | "current_user" | "session_user"
+                    | "current_database" => Some(ident.as_str()),
```

Comment:
Looking through the failing tests (specifically the `audit_log.slt`) I realized adding new aliases here is backwards incompatible, and currently has a bug. I filed an issue, https://github.com/MaterializeInc/materialize/issues/27228, and will bring this up at a team meeting, but for now do you mind remove this part of the PR?